From 9cd7a17c8a39a0b07e47467a9487b2e3c9fe5492 Mon Sep 17 00:00:00 2001
From: Takumi <takuoh@tuta.io>
Date: Mon, 2 Jan 2023 12:14:20 +0900
Subject: [PATCH] force use ja_JP.UTF-8

---
 PKGBUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/PKGBUILD b/PKGBUILD
index dbbf046..fffc3c5 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -20,6 +20,7 @@ prepare() {
     cat script/configure.perl.org                |
     sed '/until/i$ans = "y";'                    |
     sed "/usr[/]share[/]man[/]/s@/@${pkgdir}/@1" > script/configure.perl
+    sed -i 's/$LANG/ja_JP.UTF-8/g' script/configure.perl
 }
 
 build() {
@@ -28,7 +29,6 @@ build() {
 }
 
 package() {
-    mkdir -p ${pkgdir}/usr/share/man/${LANG}
     cd ${srcdir}/${pkgname}-${pkgver}
     make install
 }
-- 
2.39.0

