From 5f83352238419f1251fd3c2146837e44e3a6cc8e Mon Sep 17 00:00:00 2001
From: Takumi <takuoh@tuta.io>
Date: Mon, 2 Jan 2023 12:34:40 +0900
Subject: [PATCH] Add suid bit for /opt/visual-studio-code/chrome-sandbox

---
 PKGBUILD | 1 +
 1 file changed, 1 insertion(+)

diff --git a/PKGBUILD b/PKGBUILD
index 01ee106..043c9e7 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -69,5 +69,6 @@ package() {
 
   # Launcher
 	install -m755 "${srcdir}/${_pkgname}-bin.sh" "${pkgdir}/usr/bin/code"
+        chmod 4755 $pkgdir/opt/visual-studio-code/chrome-sandbox
 }
 
-- 
2.39.0

