diff --git a/PKGBUILD b/PKGBUILD
index 2b35fc0..7cd7e15 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -46,7 +46,7 @@ package() {
 
   # Create starter script for discord
   echo "#!/bin/sh" > "$srcdir"/$_pkgname
-  echo "exec electron16 /usr/lib/$_pkgname/app.asar \$@" >> "$srcdir"/$_pkgname
+  echo "exec electron16 --enable-accelerated-mjpeg-decode --enable-accelerated-video --ignore-gpu-blacklist --enable-native-gpu-memory-buffers --enable-gpu-rasterization /usr/lib/$_pkgname/app.asar \$@" >> "$srcdir"/$_pkgname
 
   install -d "$pkgdir"/usr/{bin,share/{pixmaps,applications}}
   install -Dm 755 $_pkgname "$pkgdir"/usr/bin/$_pkgname
