diff --git a/PKGBUILD b/PKGBUILD
index 3f8cea6..e5969a5 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -22,7 +22,7 @@ for external in "${!externals[@]}"; do
 done
 
 arch=(x86_64 i386 armv7h x86_64_v3 aarch64 pentium4)
-depends=(fmt spdlog nlohmann-json systemd libelf zlib libbpf)
+depends=(libfmt.so=9 libspdlog.so=1.11 libelf zlib libbpf)
 makedepends=(cmake clang git nlohmann-json bpf)
 optdepends=("ananicy-rules-git: community rules"
             "ananicy-rules: rules for ananicy-cpp")
@@ -41,7 +41,7 @@ prepare() {
 		-DUSE_EXTERNAL_SPDLOG=ON \
 		-DUSE_EXTERNAL_JSON=ON \
 		-DUSE_EXTERNAL_FMTLIB=ON \
-		-DENABLE_SYSTEMD=ON \
+		-DENABLE_SYSTEMD=OFF \
 		-DUSE_BPF_PROC_IMPL=ON \
 		-DBPF_BUILD_LIBBPF=OFF \
 		-DVERSION=${_pkgver}
