diff --git a/PKGBUILD b/PKGBUILD
index 4dd60e0..c40c605 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,7 +3,7 @@
 
 pkgbase=foot
 pkgdesc='Fast, lightweight, and minimalistic Wayland terminal emulator'
-pkgname=($pkgbase $pkgbase-terminfo)
+pkgname=($pkgbase)
 pkgver=1.15.1
 pkgrel=1
 url="https://codeberg.org/dnkl/$pkgbase"
@@ -36,6 +36,7 @@ build() {
 		. build \
 		-Dterminfo=disabled \
 		--prefix=/usr \
+                -Dime=false \
 		--wrap-mode=nodownload
 	sed 's/@default_terminfo@/foot-extra/g' foot.info |
 		tic -x -o build -e foot-extra,foot-extra-direct -
@@ -63,10 +64,3 @@ package_foot() {
 	install -Dm0644 -t "$pkgdir/usr/share/licenses/$pkgname/" LICENSE
 }
 
-package_foot-terminfo() {
-	pkgdesc="Extra non-standard terminfo files for $pkgbase, a Wayland terminal emulator"
-	depends=(ncurses)
-	cd "$pkgbase"
-	install -Dm0644 -t "$pkgdir/usr/share/terminfo/f/" build/f/foot-extra*
-	install -Dm0644 -t "$pkgdir/usr/share/licenses/$pkgname/" LICENSE
-}
